### ISSUE TITLE
How to replace ignore_user_agent functionality?

### ISSUE THREAD

**Issue by jschuman** [airbrake/airbrake-ruby#9]
Migrating from 4.3.4 where we used the `ignore_user_agent` configuration option.  I'm trying to figure out how to replicate that functionality using the new `add_filter` feature.  Any guidance?

**Comment by kyrylo** [comment]
Thanks for trying out the new Airbrake gem and bug reports! :)

That would be easy (assuming [this fix](https://github.com/airbrake/airbrake-ruby/pull/10) is merged in and the new gem is published):

```ruby
Airbrake.add_filter do |notice|
  notice.ignore! if notice[:context][:userAgent].match(/curl/)
end
```

In the meantime, you can use the class `add_filter` API.

**Comment by jschuman** [comment]
Excellent!  Thank you for the prompt reply!

**Comment by kyrylo** [comment]
We've just released airbrake-ruby-1.0.1, so you can use both APIs without restricitons. Happy Airbraking!
